### PR TITLE
Add Eth plugins blogpost

### PR DESCRIPTION
**Pull request by pscott**

### CODE REVIEW

File: _posts/2021-01-07-ethereum-plugins.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: Ethereum Smart Contract plugins
+author: jean
+summary: A new frontier for the security of human interactions with smart contracts
+featured-img: puzzle-eth
```
Comment (lcastillo-ledger) [inline _posts/2021-01-07-ethereum-plugins.md:6]: missing corresponding pictures. Title pictures goes under assets/img/posts

File: _includes/signatures/jean.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<div class="post-signature">
+  <div class="post-signature__prepend">
+    <img class="post-signature-prepend__picture" alt="post-signature-picture" src="/assets/img/signatures/jean.jpg" />
```
Comment (lcastillo-ledger) [inline _includes/signatures/jean.html:3]: missing Jean's nice face :) in /assets/img/signatures/jean.jpg

File: _posts/2021-01-07-ethereum-plugins.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+categories: Tech
+---
+
+# Ethereum plugins
+
+_A new frontier for the security of human interactions with smart contracts_
+
+## Smart Contracts?
+
+Unlike with Bitcoin, where most transactions are simple value transfers from an address to another, Ethereum transactions often contains interactions with `Smart Contracts`, which are programs running on its blockchain. Every time someone lends coins on [`Compound`](https://compound.finance/), swaps some tokens on [`Paraswap`](https://paraswap.io/) or pets his [`Cryptokitties`](https://www.cryptokitties.co/), smart contract transactions are involved. Supporting these use cases on hardware wallets is essential to sustain the growth of the Ethereum ecosystem without making compromises on its security.
```
Comment (lcastillo-ledger) [inline _posts/2021-01-07-ethereum-plugins.md:16]: s/from an address to another/from one address to another/
s/making compromises on its security/making compromise on its security/

File: _posts/2021-01-07-ethereum-plugins.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+## Previous state of smart contract support
+
+Until now, the device could not provide users with secure display when interacting with smart contracts. One could interact with any smart contract, but the device would just warn the user that "some data was present", and the user would have to blindly accept to sign the data, without being able to double-check that the data was valid.
```
Comment (lcastillo-ledger) [inline _posts/2021-01-07-ethereum-plugins.md:20]: s/with secure display/with a secure display/
s/that the data was valid/the validity of said data/

File: _posts/2021-01-07-ethereum-plugins.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+
+- It required too much knowledge of the Ethereum app’s codebase for third party developers to be interested.
+- Even if they did, adding full support for many smart contracts would not scale, both in term of flash space used and technical debt added to the application parsing logic.
+
+## Introducing Ethereum plugins:
+
+With the release of `Ethereum app v1.8.5`, we are laying the foundation to make adding support for a full smart contract on Ledger products much easier and scalable.
+To do so, the Ethereum transaction parser is now hookable by a new kind of applications: plugins.
+Plugins are small applications dedicated to parsing custom transaction fields and building a custom display to show on screen to the user. They are lightweight apps that can be installed on the deivce, just like any other app. Users only need to install the plugins for the smart contracts they plan to interact with. This makes supporting smart-contracts much more scalable: where previously the ethereum app would have been bloated with all sorts of different smart-contracts support, now the code logic for every smart contract is stored on different plugins, and users simply install what they need!
```
Comment (lcastillo-ledger) [inline _posts/2021-01-07-ethereum-plugins.md:41]: s/on the deivce/on the device/
s/smart-contracts/smart contracts/ x2

File: _posts/2021-01-07-ethereum-plugins.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+</figure>
+</center>
+
+There are only a few plugins available at the time of this writing ([`Paraswap`](https://github.com/LedgerHQ/app-plugin-paraswap)), but we hope to have plugins available for every major Ethereum smart contract in the near future: this depends on Ledger, but also on Ledger's community. By lowering the entry bar to adding smart contract support, we hope to get traction from smart contract programmers so they too, embrace the security of hardware wallets.
+
+## Case overview: the Compound plugin
+
+Compound is a well known set of DeFi smart contracts allowing people to lend and borrow cryptocurrencies in a trustless fashion on the Ethereum blockchain. We recently added support for Compound lending operation and we’re going to present a schematized overview of the flow of a lending transaction processing assisted by a plugin.
+
+![compound plugin interactions schema](../assets/eth_plugins/compound-plugin-interactions-schema.png "Overview of a lending transaction processng using a plugin")
```
Comment (lcastillo-ledger) [inline _posts/2021-01-07-ethereum-plugins.md:57]: On the schema:
* Forge a transaction might not be the right word, it implies creating a fake
* Loop over ProvideParameter?
* s/Copy parameter as the amount/Copy parameter such as the amount/

**Review by lcastillo-ledger — CHANGES_REQUESTED** [review]

File: _posts/2021-01-07-ethereum-plugins.md (hunk 6 above)
Comment (TamtamHero) [reply _posts/2021-01-07-ethereum-plugins.md:57]: For point 2 and 3: we're in the specific case of a Compound lending transaction: there's only 1 parameter, which is the amount

File: _posts/2021-01-07-ethereum-plugins.md [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+## Supporting all smart contracts used to be hard
+
+Adding full support for any given smart contract available in the wild used to be a hard task for many reasons:
+
+- It required too much knowledge of the Ethereum app’s codebase for third party developers to be interested.
+- Even if they did, adding full support for many smart contracts would not scale, both in term of flash space used and technical debt added to the application parsing logic.
+
+## Introducing Ethereum plugins:
+
+With the release of `Ethereum app v1.8.5`, we are laying the foundation to make adding support for a full smart contract on Ledger products much easier and scalable.
```
Comment (pdesreux-ledger) [inline _posts/2021-01-07-ethereum-plugins.md:39]:
```suggestion
With the release of `Ethereum app v1.8.5`, we are laying the foundation to make support addition for a full smart contract on Ledger products much easier and scalable.
```

**Review by pdesreux-ledger — COMMENTED** [review]
Grammar and spell check

**Pull request merged**